src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/__init__.py

```diff
@@ -12,21 +12,33 @@
 # pylint: disable=line-too-long
 class MonitorArgumentContext(AzArgumentContext):
 
-    def resource_parameter_context(self, dest, arg_group=None, required=True):
+    def resource_parameter_context(self, dest, arg_group=None, required=True, skip_validator=False):
         from azure.cli.command_modules.monitor.validators import get_target_resource_validator
-        self.argument(dest, options_list='--resource', arg_group=arg_group, required=required, validator=get_target_resource_validator(dest, required), help="Name or ID of the target resource.")
-        self.extra('namespace', options_list='--resource-namespace', arg_group=arg_group, help="Target resource provider namespace.")
-        self.extra('parent', options_list='--resource-parent', arg_group=arg_group, help="Target resource parent path, if applicable.")
-        self.extra('resource_type', options_list='--resource-type', arg_group=arg_group, help="Target resource type. Can also accept namespace/type format (Ex: 'Microsoft.Compute/virtualMachines)')")
-        self.extra('resource_group_name', options_list=('--resource-group', '-g'), arg_group='Target Resource')
+        self.argument(dest, options_list='--resource', arg_group=arg_group, required=required,
+                      validator=get_target_resource_validator(dest, required) if not skip_validator else None,
+                      help="Name or ID of the target resource.")
+        self.extra('namespace', options_list='--resource-namespace', arg_group=arg_group,
+                   help="Target resource provider namespace.")
+        self.extra('parent', options_list='--resource-parent', arg_group=arg_group,
+                   help="Target resource parent path, if applicable.")
+        self.extra('resource_type', options_list='--resource-type', arg_group=arg_group,
+                   help="Target resource type. Can also accept namespace/type format "
+                        "(Ex: 'Microsoft.Compute/virtualMachines)')")
+        self.extra('resource_group_name', options_list=('--resource-group', '-g'), arg_group=arg_group)
```

Comment:
I think we expect a list rather than a tuple?
@tjprescott?

Reply:
Tuple *does* work, but yes, please replace it with the list syntax.  We really don't want anyone using the tuple syntax anymore.

Reply:
If it is used as a read only collection, there is no different between them

Reply:
What's wrong with tuple?

Reply:
When people would do this `option_list=('--foo')` it used to cause very hard to diagnose problems.  We switched to list since that works fine there. But there are other instances of tuple in the code so if I really want to get away from that I'll do it as a separate PR.

Reply:
ok.